lua/textcase/extensions/whichkey.lua

```diff
@@ -0,0 +1,28 @@
+local M = {}
+
+function M.register(mode, mappings)
+  local opts = {
+    n = {
+      mode = "n",
+      buffer = nil,
+      silent = true,
+      noremap = true,
+      nowait = true,
+    },
+    v = {
+      mode = "v",
+      buffer = nil,
+      silent = true,
+      noremap = true,
+      nowait = true,
+    }
+  }
+
+  pcall(
+    require("which-key").register,
```

Comment:
@johmsalas I think this is probably why you had issues, this function always requires which key even if it is inside a `pcall` because rather than wrapping the `require` you wrap the register.

This should be `local ok, whichkey = pcall(require, 'which-key')` then if `ok` is true then use register

Reply:
🤯 I'm realizing you are a ninja.
If I could give you a review that you'd be "empowers others"